src/styles/blocks/burger.scss

```diff
@@ -0,0 +1,45 @@
+@use "../utils/mixins" as *;
+
+.burger {
+  &::before {
+    content: "";
+    background-image: url(/images/burger.svg);
```

Comment:
Use relative paths for background-image, change `url(/images/burger,svg)` to `url('../../images/burger,svg')`

I would suggest changing it for all images in project

Reply:
thanks for your comments :). Fixed!